packages/apputils/src/dialog.tsx

```diff
@@ -1028,16 +1026,6 @@ export namespace Dialog {
       // Setup the initial class name.
       let name = 'jp-Dialog-button';
 
-      // Add the other state classes.
-      if (data.accept) {
-        name += ' jp-mod-accept';
-      } else {
-        name += ' jp-mod-reject';
-      }
-      if (data.displayType === 'warn') {
-        name += ' jp-mod-warn';
-      }
-
```

Comment:
Let's keep those for now and remove the following CSS rules in packages/apputils/style/dialog.css

```css
.jp-Dialog-button {
  overflow: visible;
}

button.jp-Dialog-button:disabled {
  opacity: 0.6;
}

button.jp-Dialog-button:focus {
  outline: 1px solid var(--jp-brand-color1);
  outline-offset: 4px;
  -moz-outline-radius: 0;
}

button.jp-Dialog-button:focus::-moz-focus-inner {
  border: 0;
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-accept:focus,
button.jp-Dialog-button.jp-mod-styled.jp-mod-warn:focus,
button.jp-Dialog-button.jp-mod-styled.jp-mod-reject:focus {
  outline-offset: 4px;
  -moz-outline-radius: 0;
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-accept:focus {
  outline: 1px solid var(--jp-accept-color-normal, var(--jp-brand-color1));
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-warn:focus {
  outline: 1px solid var(--jp-warn-color-normal, var(--jp-error-color1));
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-reject:focus {
  outline: 1px solid var(--jp-reject-color-normal, var(--md-grey-600, #757575));
}
```